### PR TITLE
convert all JSON BLOB content to strings

### LINKED ISSUE #32: JSON BLOB performs undesirable conversion of floating-point types with int-like values
### Checked for duplicates

No - I haven't checked

### 🐛 Describe the bug

See [registry-api #191](https://github.com/NASA-PDS/registry-api/issues/191)

### 🕵️ Expected behavior

I expect information to be retained

### 📜 To Reproduce

_No response_

### 🖥 Environment Info

_No response_

### 📚 Version of Software Used

_No response_

### 🩺 Test Data / Additional context

_No response_

### 🦄 Related requirements

_No response_

### ⚙️ Engineering Details

Decision has been made by @jordanpadams @tloubrieu-jpl @alexdunnjpl to extract all data from XML as string-typed, ensuring that no data is lost when parsing and writing to JSON BLOB.

### PR DESCRIPTION
**Pull request by alexdunnjpl**
## 🗒️ Summary
See title

## ⚙️ Test Data and/or Report
See branch tests

## ♻️ Related Issues
fixes #32

### CODE REVIEW

File: src/main/java/gov/nasa/pds/registry/common/meta/FileMetadataExtractor.java [hunk 1]
```diff
@@ -240,7 +240,7 @@ public static String getJsonBlob(File file) throws Exception
         try
         {
             source = new FileReader(file);
-            JSONObject json = XML.toJSONObject(source);
+            JSONObject json = XML.toJSONObject(source, true);
```
Comment (alexdunnjpl) [inline src/main/java/gov/nasa/pds/registry/common/meta/FileMetadataExtractor.java:243]: `true`, here, is parameter `keepStrings`

**Review by tloubrieu-jpl — COMMENTED** [review]
Thanks @alexdunnjpl Have you tried to test that in the docker compose environment ? That would provide us with an integration test. I will try that later today, or we can have a tag up to figure how to do that together.

**Comment by alexdunnjpl** [comment]
@tloubrieu-jpl I haven't - I'd assumed that any necessary tests would be included in the branch integration tests.

If the expectation is that all PRs touching registry are tested through a manual process using the dockerized registry, it would probably be helpful for me to crib your workflow for that, yes.

**Review by tloubrieu-jpl — APPROVED** [review]
That works ! The integration tests partly failed because it was a bit more complicated than the manual procedure that we initially proposed during the breakout today, but if we interpret the test results, that work !

**Comment by alexdunnjpl** [comment]
Thanks Thomas!

**Pull request merged**